src/components/Popup.js

```diff
@@ -33,6 +34,10 @@ export default class Popup {
     }
   }
 
+  renderLoading(buttonText) {
```

Comment:
`renderLoading` can't be in `Popup` because the image modal doesn't have any submit button. Please, move it into `PopupWithForm.js`